### ISSUE TITLE
fuseops.ReadFileOp error: Backend Error

### ISSUE THREAD

**Issue by sniklaus** [GoogleCloudPlatform/gcsfuse#408]
Any suggestions on how to deal with the following backend error? Thanks!

```
fuseops.ReadFileOp error: fh.reader.ReadAt: startRead: NewReader: googleapi: got HTTP response code 503 with body: Backend Error
```

Please also see: #407, #410

**Comment by bjornleffler** [comment]
I'm looking at this. Thanks for reporting.

**Comment by sniklaus** [comment]
Seems to be fixed with #414, thanks!
